src/Compilers/VisualBasic/Test/Emit/Emit/EmitCustomModifiers.vb

```diff
@@ -107,5 +108,188 @@ End Class
 
             CompileAndVerify(comp, expectedOutput:=<![CDATA[2]]>)
         End Sub
+
+        <Fact>
+        Public Sub NonExtensibleReadOnlySignaturesAreRead()
```

Comment:
>Public Sub NonExtensibleReadOnlySignaturesAreRead() [](start = 8, length = 51)

Consider moving all these tests into a new file dedicated to ReadonlyRef feature.